packages/react/src/feature-link-list/FeatureLinkListItem.tsx

```diff
@@ -0,0 +1,89 @@
+import { ReactNode } from 'react';
+import { LinkProps, packs, useId } from '../core';
+import { TextLink, TextLinkExternal } from '../text-link';
+import { Box, linkStyles } from '../box';
+import { Flex } from '../flex';
+import { Stack } from '../stack';
+import { Text } from '../text';
+import { ArrowRightIcon } from '../icon';
+import { FeatureLinkListBackground, hoverColorMap } from './utils';
+
+export function useLinkDescriptionId(idProp?: string) {
+	const autoId = useId(idProp);
+	const sanitizeId = autoId?.replace(/[^a-zA-Z0-9-_]/g, '_'); //sanitize id to have valid html test pass
```

Comment:
This sanitize step isn't suitable here because it deviates from the rest of the design system. If you believe switching the delimiter from `:` to `_` is a better solution I'd encourage you to open a separate PR that targets the `useId` utility in the `Core` package. AFAICT the current solution isn't actually an issue for us because we don't use CSS id selectors. We only use IDs in html attributes and links.

Reply:
This is on me for not reviewing this commit. I incorrectly assumed the import had just been changed to be from core

Reply:
It's not ideal and we may update our useId hook, but the rest of the DS has this in component tests:

```
expect(container).toHTMLValidate({
	extends: ['html-validate:recommended'],
	rules: {
		// react 18s `useId` break this rule
		'valid-id': 'off',
	},
});
```

html-validate incorrectly whinges at this. The real W3C validator doesn't care